### PR TITLE
Feature/camelize cli options

### PR DESCRIPTION
**Pull request by tommmyy**

### CODE REVIEW

**Comment by wafflepie** [comment]
I'll take a look at this later in the day. Looks very promising! :)

File: packages/react-union-scripts/README.md [hunk 1]
```diff
@@ -121,24 +116,119 @@ yarn test --release
...
+
+## `union.config.js`
+
+Configuration can eiher exports:
+- static JSON object or
+- function.
+
+To the function is passed object that describes flags derived from calling our CLI api:
+
+```
```
Comment (wafflepie) [inline packages/react-union-scripts/README.md]: Replace with ```js

File: packages/react-union-scripts/README.md [hunk 2]
```diff
@@ -148,43 +238,59 @@ File exports objects with following properties.
...
+			name: 'MyFirstApp',
+			proxy: { port: 5000 },
 		},
+		{ name: 'MySecondApp' },
 	],
 };
 ```
 
-MyApp uses port `3330` for proxy and MySecondApp uses `3333`.
+will MyFirstApp use proxy port `5000` and MySecondApp will use common value `3333`.
```
Comment (wafflepie) [inline packages/react-union-scripts/README.md]: Swap the first two words, please.

File: boilerplates/react-union-boilerplate-basic/public/SampleApp/index.ejs [hunk 3]
```diff
@@ -1,28 +1,21 @@
...
-  <script data-union-widget="content" data-union-container="content" type="application/json">
-    {
-      "something": "bar"
-    }
-  </script>
-</body>
-
-</html>
+    <div id="content"></div>
+    <script data-union-widget="content" data-union-container="content" type="application/json"></script>
```
Comment (wafflepie) [inline boilerplates/react-union-boilerplate-basic/public/SampleApp/index.ejs]: Don't we want to showcase passing of initial data?

**Comment by tommmyy** [comment]
@wafflepie I will address and correct the issues either today or close to end of the week.
I wanna add some stuff to simplify the union.config.js even more.

Following should work:

```js
module.exports = ["app1", "app2"]
```
```js
module.exports = { templateFilename: "start.ejs", aps: ["app1", "app2"] }
```
and if the union.config.js is not specified at all, than the names of the aps should be interfere from the names of the dis in the default `paths.public` directory.

If we follow this logic we do not need config at all.

File: package.json [hunk 4]
```diff
@@ -45,12 +47,18 @@
...
   ],
   "scripts": {
-    "bootstrap": "yarn config set workspaces-experimental true && lerna bootstrap",
+    "bootstrap": "lerna bootstrap",
     "example:basic": "cd ./boilerplates/react-union-boilerplate-basic && yarn start",
     "lint": "eslint \"packages/**/*.js\" \"boilerplates/**/*.js\" --cache",
     "lint:fix": "yarn lint --fix",
-    "precommit": "yarn lint:fix",
-    "prepush": "yarn lint:fix"
+    "pretty": "prettier --write ./packages/**/*.js ./boilerplates/**/*.js && yarn lint:fix",
```
[the remark this answers is not in the document]
Comment (tommmyy) [reply package.json]: Truth, plugin do that automagically.

File: packages/react-union-scripts/README.md (hunk 1 above)
Comment (tommmyy) [reply packages/react-union-scripts/README.md]: ok

File: packages/react-union-scripts/README.md (hunk 2 above)
Comment (tommmyy) [reply packages/react-union-scripts/README.md]: typo

File: boilerplates/react-union-boilerplate-basic/public/SampleApp/index.ejs (hunk 3 above)
Comment (tommmyy) [reply boilerplates/react-union-boilerplate-basic/public/SampleApp/index.ejs]: Can be done in another issue. I will create it.

**Comment by tommmyy** [comment]
@wafflepie Can you check it?

File: package.json [hunk 5]
```diff
@@ -51,13 +51,13 @@
     "example:basic": "cd ./boilerplates/react-union-boilerplate-basic && yarn start",
     "lint": "eslint \"packages/**/*.js\" \"boilerplates/**/*.js\" --cache",
     "lint:fix": "yarn lint --fix",
-    "pretty": "prettier --write ./packages/**/*.js ./boilerplates/**/*.js && yarn lint:fix",
+    "pretty": "prettier --write",
```
Comment (wafflepie) [inline package.json]: Do you insist on keeping the script there? I personally never use Prettier alone, only with ESLint, so I'd remove this line completely.

**Comment by tommmyy** [comment]
Done and merged

**Comment by jamescoq** [comment]
Nice work :)

**Pull request merged**